packages/ses/src/error/assert.js

```diff
@@ -568,5 +568,15 @@ const assert = makeAssert();
 export { assert };
 
 // Internal, to obviate polymorphic dispatch, but may become rigorously
-// consistent with @endo/error.
-export { makeError, note as annotateError, redactedDetails as X, quote as q };
+// consistent with @endo/error:
+
+/** @type {AssertionFunctions['typeof']} */
+const assertTypeof = assert.typeof;
+
+export {
+  assertTypeof,
+  makeError,
+  note as annotateError,
+  redactedDetails as X,
+  quote as q,
+};
```

Comment:
Is this a drive-by?

Reply:
No, this was my preferred solution to avoid a polymorphic dispatch at `assertTypeof` elsewhere in this change. You had good feedback elsewhere which will change this to `assertEqual` if not a more systematic change.